Source/cmake/OptionsWPE.cmake

```
@@ -158,6 +160,10 @@ include(GStreamerDependencies)
 
 WEBKIT_OPTION_END()
 
+if (NOT ENABLE_WPE_PLATFORM AND NOT ENABLE_WPE_LEGACY_API)
+    message(FATAL_ERROR "WPE Platform and Legacy API cannot be disabled at the same time")
```

Small nit: I think it is easier to parse if we worded this in a positive mode, and indicating which CMake option to toggle. For example, this could be:


```suggestion
    message(FATAL_ERROR "At least one of ENABLE_WPE_PLATFORM or ENABLE_WPE_LEGACY_API needs to be enabled")
```